components/progresscircle/index.css

```diff
@@ -68,591 +68,56 @@
-	border-width: var(--mod-progress-circle-thickness, var(--spectrum-progress-circle-thickness));
-	border-radius: var(--mod-progress-circle-size, var(--spectrum-progress-circle-size));
-	border-color: var(--mod-progress-circle-track-border-color, var(--spectrum-progress-circle-track-border-color));
-}
+	&.spectrum-ProgressCircle--staticWhite {
+		--spectrum-progress-circle-track-border-color: var(--spectrum-static-white-track-color);
+		--spectrum-progress-circle-fill-border-color: var(--spectrum-static-white-track-indicator-color);
+	}
 
-/* circle progress fill parent container */
-.spectrum-ProgressCircle-fills {
-	position: absolute;
-	inset-block-start: 0;
-	inset-inline-start: 0;
-	inline-size: 100%;
-	block-size: 100%;
+	&.spectrum-ProgressCircle--staticBlack {
+		--spectrum-progress-circle-track-border-color: var(--spectrum-static-black-track-color);
+		--spectrum-progress-circle-fill-border-color: var(--spectrum-static-black-track-indicator-color);
+	}
 }
 
-/* circle progress filled */
+.spectrum-ProgressCircle-track,
 .spectrum-ProgressCircle-fill {
 	box-sizing: border-box;
 	inline-size: var(--mod-progress-circle-size, var(--spectrum-progress-circle-size));
 	block-size: var(--mod-progress-circle-size, var(--spectrum-progress-circle-size));
-
-	border-style: solid;
-	border-width: var(--mod-progress-circle-thickness, var(--spectrum-progress-circle-thickness));
-	border-radius: var(--mod-progress-circle-size, var(--spectrum-progress-circle-size));
-	border-color: var(--highcontrast-progress-circle-fill-border-color, var(--mod-progress-circle-fill-border-color, var(--spectrum-progress-circle-fill-border-color)));
 }
 
-/* circle colors over background */
-.spectrum-ProgressCircle.spectrum-ProgressCircle--staticWhite {
-	.spectrum-ProgressCircle-track {
-		border-color: var(--mod-progress-circle-track-border-color-over-background, var(--spectrum-progress-circle-track-border-color-over-background));
-	}
-
-	.spectrum-ProgressCircle-fill {
-		border-color: var(--highcontrast-progress-circle-fill-border-color-over-background, var(--mod-progress-circle-fill-border-color-over-background, var(--spectrum-progress-circle-fill-border-color-over-background)));
-	}
-}
-
-/* animation masks - circle fill progress */
-.spectrum-ProgressCircle-fillMask1,
-.spectrum-ProgressCircle-fillMask2 {
-	inline-size: 50%;
-	block-size: 100%;
-	transform-origin: 100% center;
-	transform: rotate(180deg);
-	overflow: hidden;
-	position: absolute;
-}
-
-.spectrum-ProgressCircle-fillSubMask1,
-.spectrum-ProgressCircle-fillSubMask2 {
-	inline-size: 100%;
-	block-size: 100%;
-	transform-origin: 100% center;
-	overflow: hidden;
-	transform: rotate(-180deg);
+.spectrum-ProgressCircle-track {
+	stroke-width: var(--mod-progress-circle-thickness, var(--spectrum-progress-circle-thickness));
+	stroke: var(--highcontrast-progressbar-track-color, var(--mod-progress-circle-track-border-color, var(--spectrum-progress-circle-track-border-color)));
```

Comment:
I think we need to delete this progress bar custom property in favor of a `--highcontrast-progress-circle-track-border-color` instead.